### ISSUE TITLE
Why not use the cls_token instead of average pooling in BeiT ?

### ISSUE THREAD

**Issue by JosephChenHub** [microsoft/unilm#820]
As claimed in the section 2.2 of BeiT,  "Moreover, we prepend a special token [S] to the input sequence.''
But at the finetuning stage,   "Specifically, we use average pooling to aggregate the representations, and feed the global to a softmax classifier. " The implementation of BeiT also shows that BeiT uses average pooling to aggregate the final outputs for image classification instead of using the hidden output corresponding to the cls_token ,  so I have some questions
```
1. "a specical token [S]" is indeed the "cls_token" according to the code and the paper, but it's meaningless due to the average pooling.
2.  "cls_token" vs. average pooling, which is better?  or just say both are ok due to the powerful transformer arch. 
```

**Comment by donglixp** [comment]
Avg pooling tends to perform better according to empirical studies.
